src/CowSdk.ts

```diff
@@ -1,30 +1,54 @@
+import { Signer } from 'ethers'
+import log, { LogLevelDesc } from 'loglevel'
 import { version as SDK_VERSION } from '../package.json'
 import { CowApi } from './api'
+import { CowError } from './utils/common'
 import { SupportedChainId as ChainId } from '/constants/chains'
 import { validateAppDataDocument } from '/utils/appData'
 import { Context, CowContext } from '/utils/context'
 import { signOrder, signOrderCancellation, UnsignedOrder } from '/utils/sign'
 
+log.setDefaultLevel('debug')
+
+type Options = {
+  loglevel?: LogLevelDesc
```

Comment:
here you implemented something in addition. It's not just the chainId. Is it possible the base branch is incorrect?